tmt-app/src/Styling/CustomStyling.js

```diff
@@ -0,0 +1,94 @@
+import {styled,Button,Container, createTheme} from '@mui/material';
+
+export const CustomTheme = createTheme({
+    palette: {
+        BTMilkTea: {
+            main:'#F2DAC4'
+        },
+        Lychee: {
+            main:'#F6A794'
+        },
+        ThaiTea: {
+            main:'#E0A878'
+        },
+        Matcha: {
+            main:'#73956F'
+        },
+        RedBean: {
+            main:'#732C2C'
+        },
+        BobaHighlight: {
+            main:'#616587'
+        },
+        InactiveGrey: {
+            main:'#5B5D6C'
+        },
+        Boba: {
+            main:'#1B1D2E'
+        },
```

Comment:
nice, really happy to see we got these set up, do we need to also override the theme preset colors?

https://mui.com/material-ui/customization/theming/#api

I think we can address this maybe in a separate ticket, feels like it could be quite a bit of testing to make sure we cover enough colors, possibly also communicating with design

I dont think this is a blocker for this PR either, just want to point this out